### DISCUSSION TITLE
Static worker unexpectedly exited with code: null and signal: SIGTERM

### DISCUSSION THREAD

**Discussion by LeVuMinhHuy** [vercel/community#3801] [Help] [answered]
### Summary

When I run pnpm build locally, it builds successfully without any errors. However, when I deploy it on Vercel, it keeps failing and returns this error:

Static worker unexpectedly exited with code: null and signal: SIGTERM

I'm pretty sure i don't have .next folder in my repository.

This is my next.config.js

```javascript
/** @type {import('next').NextConfig} */
const nextConfig = {
  webpack: (config) => {
    config.module.rules.push({
      test: /\.node$/,
      use: "node-loader",
    });

    return config;
  },
  images: {
    domains: ["https://firebasestorage.googleapis.com/"],
  },
};

module.exports = nextConfig;
```

I really need some help. I have no idea why this is happening

![image](https://github.com/vercel/community/assets/36370320/771793bc-696b-45c4-8ae6-80cc8170fc7f)

### Steps to Reproduce

run `pnpm run build` on vercel

**Comment by amyegan** [comment]
Hey @LeVuMinhHuy. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then someone here may be able to recognize what is going wrong.

**Reply by LeVuMinhHuy** [reply, marked as answer]
I increased the `staticPageGenerationTimeout` to 180 and it's working now, thank you @amyegan
